doc/Programming-guide/Key-platform-features/Data-collection/index.md

```diff
@@ -226,128 +226,100 @@ The log record header schema described below:
-configure Couchbase servers list, bucket name and password.
+### Confirm delivery option
 
-* ### [File system log appender]({{root_url}}Programming-guide/Key-platform-features/Data-collection/File-system-log-appender)
+Every [Kaa client]({{root_url}}Glossary/#kaa-client) stores logs in a special log storage before sending them to the [Kaa server]({{root_url}}/Glossary/#kaa-server) node.
+By default, when a log appender on the Kaa server side receives a log from the Kaa client, it will send a delivery confirmation back to the Kaa client.
+In case of a successful delivery, the Kaa client deletes the log copies from its local storage.
+In case of a delivery error, Kaa client receives the error code and either sends the log to the same server node again or sends it to a different node.
+The delivery confirmation option makes the log appender verify that the received log has been recorded in the log storage prior to sending the confirmation.
 
-The file system log appender stores received logs into the local file system of the Operations service. This log appender may be used for test purposes
-or in pair with tools like Flume and others. You can configure logs root path, rolling file name pattern, rolling max history, trigger max file size and
-[other]({{root_url}}Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/#configuration) parameters for this log appender.
+The following scenario illustrates how the delivery confirmation works:
 
-* ### [Flume log appender]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Flume-log-appender)
+1. You set up a log appender with the **Confirm delivery** option enabled.
+The Kaa client will receive a message about successful logs delivery only after the Kaa node successfully receives and writes the logs to external storage.
+If the Kaa node fails to write the logs to the storage, the Kaa client will send the logs again.
+If there are more than one log appender with the **Confirm delivery** option enabled, the Kaa client will receive a message about successful logs delivery only after each log appender successfully writes its logs to the storage.
 
-The Flume log appender encapsulates received logs into Flume events and sends these events to external Flume sources via Avro RPC. You can configure
-flume event format, hosts balancing type, include client-side and/or server-side profile data and
-[other]({{root_url}}Programming-guide/Key-platform-features/Data-collection/Flume-log-appender/#configuration) parameters for this log appender.
+2. You set up a log appender with the **Confirm delivery** option disabled.
+The Kaa client will receive a message about successful logs delivery but if the Kaa node fails to write the logs to the external storage, the Kaa client will not be informed about the failure and will not try to send the logs again.
```

Comment:
Please use this description: "The Kaa client will receive a message about successful logs delivery to the Kaa node but if the Kaa node ..."